07_modules_libs/homework/handler.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi/v5"
+)
+
+type HandleReq struct {
+	Data interface{}
+}
+
+func (h *HandleReq) Get(w http.ResponseWriter, r *http.Request) {
+	const API_KEY = "HXPQ9N2CBVG2W6MJ"
+	ticker := chi.URLParam(r, "ticker")
+	date := chi.URLParam(r, "date")
+	url := fmt.Sprintf("https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=%s&apikey=%s", ticker, API_KEY)
+
+	response, err := http.Get(url)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Log.Fatal will exit the process with non zero code. The best way is write header with status code  eg 500 and return from handler func.